README.md

```diff
@@ -0,0 +1,9 @@
+Install dependencies of your subgraph and run the Graph CLI code generation:
+yarn && yarn codegen
+
+For auth run (get the access token from thegraph.com):
+graph auth https://api.thegraph.com/deploy/ <ACCESS_TOKEN>
+
+yarn build
+
+yarn deploy
```

Comment:
I am getting a subgraph validation error: [EthereumNetworkRequired]
when using yarn deploy. Please help update the readme @JasoonS